apps/pom.xml

```diff
@@ -650,6 +660,16 @@ checkJavaFormat
 			</plugins>
 		</pluginManagement>
 		<plugins>
+			<!-- Use the versions-maven-plugin plugin to help ensure that checking
+							for new versions to update dependencies to don't include alphas. -->
```

Comment:
can you include the command line and also make a note that it needs to be run from the apps directory

Reply:
Updated